XCUITests/TabCounterTests.swift

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class TabCounterTests: BaseTestCase {
+
+    func testTabIncrement() throws {
+        navigator.nowAt(NewTabScreen)
+        navigator.performAction(Action.CloseURLBarOpen)
+        waitForTabsButton()
+
+        var tabsOpen = app.buttons["Show Tabs"].value
+        XCTAssertEqual("1", tabsOpen as? String)
+
+        navigator.createNewTab()
+        navigator.nowAt(NewTabScreen)
+        navigator.performAction(Action.CloseURLBarOpen)
+        waitForTabsButton()
+
+        tabsOpen = app.buttons["Show Tabs"].value
+        XCTAssertEqual("2", tabsOpen as? String)
+    }
+
+    func testTabDecrement() throws {
+        navigator.nowAt(NewTabScreen)
+        navigator.performAction(Action.CloseURLBarOpen)
+        waitForTabsButton()
+
+        var tabsOpen = app.buttons["Show Tabs"].value
```

Comment:
I should have realized about that before and explain the purpose of the test case better...apologies... the problem where we have seen regressions is with the counter in the tab tray, not the counter in the homescreen.
This check is verifying the counter in the homescreen but the one we would like to check is this one in the tab tray
![Sin título](https://user-images.githubusercontent.com/1897507/134302620-7fd8aaf4-0a86-45b3-a665-e734581754cd.png)
:

And same for all the other checks

Reply:
Added this along the original check. The button's only label is the counter number itself so it's a little messy in the way it's accessed if that's ok.